utils/bluez-tools/files/bt-agent.cfg

```diff
@@ -0,0 +1,5 @@
+# format of pin file is: MAC_ADDR PIN
```

Comment:
Can we convert this file generation to uci
```
config device
    option mac 'xx:xx:xx:xx:xx:xx'
    option pin '1234'
```

Reply:
probably yes but as I am not a 100% familiar with openWrt and this is my first contribution I am not sure how to proceed here. Do you have some manual or a package name where this is done so I have a guide how to do this?

Reply:
done